pkg/cmd/issue/view/presentation_issue.go

```diff
@@ -0,0 +1,122 @@
+package view
+
+import (
+	"fmt"
+	"slices"
+	"strings"
+	"time"
+
+	"github.com/cli/cli/v2/api"
+	prShared "github.com/cli/cli/v2/pkg/cmd/pr/shared"
+	"github.com/cli/cli/v2/pkg/iostreams"
+)
+
+type PresentationIssue struct {
+	Title          string
+	Number         int
+	CreatedAt      time.Time
+	Comments       api.Comments
+	Author         string
+	State          string
+	StateReason    string
+	Reactions      string
+	AssigneesList  string
+	LabelsList     string
+	ProjectsList   string
+	MilestoneTitle string
+	Body           string
+	URL            string
+}
+
+// Creates a new PresentationIssue from an api.Issue.
+// The PresentationIssue is what the issue printers need to display an
+// issue to the user.
+func MapApiIssueToPresentationIssue(issue *api.Issue, colorScheme *iostreams.ColorScheme) (PresentationIssue, error) {
+	presentationIssue := PresentationIssue{
+		Title:         issue.Title,
+		Number:        issue.Number,
+		CreatedAt:     issue.CreatedAt,
+		Comments:      issue.Comments,
+		Author:        issue.Author.Login,
+		State:         issue.State,
+		StateReason:   issue.StateReason,
+		Reactions:     prShared.ReactionGroupList(issue.ReactionGroups),
+		AssigneesList: getAssigneeListString(issue.Assignees),
+		LabelsList:    getColorizedLabelsList(sortAlphabeticallyIgnoreCase(issue.Labels), colorScheme),
+		ProjectsList:  getProjectListString(issue.ProjectCards, issue.ProjectItems),
+		Body:          issue.Body,
+		URL:           issue.URL,
+	}
+
+	if issue.Milestone != nil {
+		presentationIssue.MilestoneTitle = issue.Milestone.Title
+	}
+
+	return presentationIssue, nil
+}
+
+func getProjectListString(projectCards api.ProjectCards, projectItems api.ProjectItems) string {
```

Comment:
I find this function name potentially misleading as it makes me think the function returns `[]string`.  I know naming is hard, only thing I can think of is `stringifyProjectItems` 🤷 

_Even that name is more forward thinking once we drop project cards_

Reply:
Noted. I've updated these accordingly. Let me know if you have any other feedback on the names